tests/data/common/non_ascii/non_ascii.py

```diff
@@ -0,0 +1,13 @@
+def func_non_ascii(argλ: float) -> None:
+    """Something.
+
+    Parameters
+    ----------
+    argλ : float
```

Comment:
Shall we ensure non-BMP characters (such as emoji like 😂) are correctly handled as well?

Reply:
Even so, non ASCII characters are valid in python args (actually have a use case for `λ` which is how I found this issue 😅) emojis are not. 

```py
In [1]: def f(😅: int):
   ...:     return 😅**2
  Input In [1]
    def f(😅: int):
          ^
SyntaxError: invalid character '😅' (U+1F605)
```